src/EnergyPlus/ThermalComfort.cc

```diff
@@ -678,9 +681,9 @@ namespace ThermalComfort {
 			RadTemp = CalcRadTemp( PeopleNum );
 			// Use mean air temp for calculating RH when thermal comfort control is used
 			if ( present( PNum ) ) {
-				RelHum = PsyRhFnTdbWPb( MAT( ZoneNum ), ZoneAirHumRat( ZoneNum ), OutBaroPress );
+				RelHum = PsyRhFnTdbWPb( MAT( ZoneNum ), ZoneAirHumRatAvgComf( ZoneNum ), OutBaroPress );
```

Comment:
I don't recall all the details from the original issue.  Perhaps this one is different because it's during the prediction for comfort control.  Do the changes as they currently stand fix the originally observed problem in the defect file?